src/remote/watcher.js

```diff
@@ -56,6 +71,9 @@ export default class RemoteWatcher {
       await this.pouch.setRemoteSeqAsync(changes.last_seq)
       log.lineBreak()
     } catch (err) {
+      if (err.message === 'Client has been revoked') {
```

Comment:
I imagine this string being hardcoded in multiple `if` statements...

Either extract it to some constant, or check the error type? (although I'm not even sure there is a reliable way to do so in js...)